packages/datadog-instrumentations/src/pg.js

```diff
@@ -130,6 +146,15 @@ function wrapQuery (query) {
         newQuery.then((res) => finish(null, res), finish)
       }
 
+      if (textPropObj.read) {
+        newQuery.on('end', () => {
+          finish(null, [])
+        })
+        newQuery.on('error', (err) => {
```

Comment:
```suggestion
        newQuery.on(errorMonitor, (err) => {
```

This is important, otherwise we influence the user's code. It would not crash the application anymore in case of an error that is not caught, if the user would not handle it. That is something we do not want.

The symbol may be imported as: `const { errorMonitor } = require('node:events');`